File: 1.HelloWorld/Go/main.go
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()

	fmt.Printf("Hello, world!\n")

	elapsed := time.Since(start)
	fmt.Printf("Execution time: %.6f seconds\n\n", elapsed.Seconds())
}

File: 1.HelloWorld/C/main.c
#include <stdio.h>
#include <time.h>

int main() {
    clock_t start, end;
    double cpu_time_used;
    start = clock();

    printf("Hello, world!");
    end = clock();
    cpu_time_used = ((double) (end - start)) / CLOCKS_PER_SEC;
    printf("Execution time: %.6f seconds.\n\n", cpu_time_used);
    return 0;
}
